.github/workflows/build.yml

```diff
@@ -17,9 +17,6 @@ jobs:
                   cache: "npm"
                   cache-dependency-path: |
                       package-lock.json
-                      packages/quick_start/package-lock.json
-                      packages/azure-kusto-data/package-lock.json
-                      packages/azure-kusto-ingest/package-lock.json
             - run: |
                   npm install -g npm@latest
```

Comment:
Does the release work now that our files are built to "dist-esm"?

Reply:
Mocha runs the typescript files directly (using ts-node), but if it works I'll close it.